From 88698f32444ee88ce66e40cf54b8ac3563342723 Mon Sep 17 00:00:00 2001
From: Sameer Nanda <snanda@chromium.org>
Date: Mon, 15 Nov 2010 13:38:40 -0800
Subject: [PATCH 8/8] Lowering hard drive idle timeout to 5 seconds from 20 seconds.

This change saves about 200mW of power on SSDs.

Change-Id: I7a7dfdbd7e264386d21da673b1906460ecf1c307

BUG= chromium-os:9180
TEST= ran vmstat & hdparm to see when the last read/write to the disk
happened and when the drive transitioned to standby state.  With this
change in place, on battery the hard drive transitioned to standby
state 5 seconds after the last read/write.

Review URL: http://codereview.chromium.org/5032002
---
 .../etc/laptop-mode/laptop-mode.conf               |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff -ruN laptop-mode-tools-1.59/etc/laptop-mode/laptop-mode.conf laptop-mode-tools-1.59/etc/laptop-mode/laptop-mode.conf
index 47a2381..5dc4b38 100644
--- laptop-mode-tools-1.59/etc/laptop-mode/laptop-mode.conf
+++ laptop-mode-tools-1.59/etc/laptop-mode/laptop-mode.conf
@@ -273,7 +273,7 @@ CONTROL_HD_IDLE_TIMEOUT=1
 # for battery and for AC with laptop mode on.
 #
 LM_AC_HD_IDLE_TIMEOUT_SECONDS=20
-LM_BATT_HD_IDLE_TIMEOUT_SECONDS=20
+LM_BATT_HD_IDLE_TIMEOUT_SECONDS=5
 NOLM_HD_IDLE_TIMEOUT_SECONDS=7200
 
 
-- 
1.7.2.3

